ikabot/config.py

```diff
@@ -6,6 +6,8 @@
 import os
 import random
 
+ikabotVersion = 'v7.0.0'
```

Comment:
The written version is different from the setup.py version, is this normal?

Reply:
You're right, I missed that entirely. It is fixed now.